### DISCUSSION TITLE
verify artifacts will be retained for specified retention-days

### DISCUSSION THREAD

**Discussion by aparry91** [community/community#37374] [Actions]
Hi, is there currently a way to view how long your artifacts will be retained for? 

I know that I can specify retention-days when using the uploadartifacts action, however, I would like at least for sanity purposes to verify that it hasn't just taken the default value of 90 and has actually been updated.

Cheers

**Comment by antonblr** [comment]
@aparry91 - I believe you can get this info from API calls, i.e. list all artifacts and check `expires_at` value:

All artifacts:
```shell
curl \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer <YOUR-TOKEN>" \
  https://api.github.com/repos/OWNER/REPO/actions/artifacts
```

Specific artifact:
```shell
curl \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer <YOUR-TOKEN>" \
  https://api.github.com/repos/OWNER/REPO/actions/artifacts/ARTIFACT_ID
```

Reference - https://docs.github.com/en/rest/actions/artifacts#list-artifacts-for-a-repository
